### PR TITLE
@eessex => Less noisy morgan logs on papertrail

### PR DESCRIPTION
**Pull request by kanaabe**
Use morgan's `tiny` option to log less on papertrail

cc @izakp!

### CODE REVIEW

File: api/index.coffee [hunk 1]
```diff
@@ -14,7 +14,7 @@ app.use cors()
 app.use helpers
 app.use bodyParser.json limit:'5mb', extended: true
 app.use bodyParser.urlencoded limit:'5mb', extended: true
-app.use morgan 'dev'
+app.use morgan(if NODE_ENV is 'development' then 'dev' else 'tiny')
```
Comment (izakp) [inline api/index.coffee]: <3 thanks @kanaabe!  Outputting necessary logging keeps our syslog servers healthy and happy ^__^

**Comment by kanaabe** [comment]
@izakp changed it to `combined` instead which gives us IP. Then, we can filter heroku instead!

**Comment by eessex** [comment]
@kanaabe this looks good to me, will be sure to read up on morgan

**Comment by izakp** [comment]
Great!

**Pull request merged**